Formula/h/hashcat.rb

```diff
@@ -37,6 +37,10 @@ class Hashcat < Formula
     depends_on "pocl"
   end
 
+  # Fix 'failed to create metal library' on macos
+  # extract from hashcat version 66b22fa, remove this patch when version released after 66b22fa
+  patch :DATA
```

Comment:
Can you add a link to the relevant commit here?

Reply:
added, but likely the second commit message doesn't follow Commit style, should I change it and force push?
And CI still goes to fail, same as before this PR: https://github.com/Homebrew/homebrew-core/actions/runs/10458104413/job/29014309789
I dont know how the first version contains `test::assert_match` pass CI. the CI runner machine is different or no GPU on it?

a test on a real mac can tell this PR makes sence

Reply:
No GPU in CI runners, I think!